README.md

```diff
@@ -135,4 +135,80 @@ iex> EctoDiff.diff(initial_pet, updated_pet)
  >}
 ```
 
+#### Returning Select Fields
+
+To limit processing and returning the fields and associations for the diff the `:select_fields` option may be provided a keyword list of desired fields.
+
+Any field not in `:select_fields` will be excluded from the diff.
+```elixir
+iex> EctoDiff.diff(initial_pet, updated_pet, select_fields: [pets: [:name, :id]])
+
+{:ok,
+  #EctoDiff<
+    struct: Pet,
+    primary_key: %{id: 2},
+    effect: :changed,
+    previous: #Pet<>,
+    current: #Pet<>,
+    changes: %{
+        name: {"Spot", "Spots"},
+    }
+  >
+}
+```
+
+The keyword list is a depth of 2 representation of the field and subfields for each selected field. If a field is given as a key, with subfields given as the values but is not provided as the appropriate value in a parent key it will be excluded.
```

Comment:
```suggestion
The keyword list is a depth of 2 representations of the field and subfields for each selected field. If a field is given as a key, with subfields given as the values but is not provided as the appropriate value in a parent key it will be excluded.
```